dev/integration_tests/ffi/lib/src/generated/aliasing_test.dart

```diff
@@ -0,0 +1,216 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This repo must only have code with the "Copyright 2014 The Flutter Authors" copyright.

Reply:
@Hixie Can I use flutter-only copyright even though the dart files's source-of-truth is under dart copyright? If so, I'd be happy to do it.